### PR TITLE
[AWS] Enable versioning S3

### PR DESCRIPTION
**Pull request by maxclaytoncary**

### CODE REVIEW

File: remediation/aws/enable_versioning_s3/bot.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import botocore
+import logging
+import sonrai.platform.aws.arn
+
+def run(ctx): 
+    s3_client = ctx.get_client().get('s3')
+
+    arr = ctx.resource_id.split(':')
```
Comment (wei-shang-sonrai) [inline remediation/aws/enable_versioning_s3/bot.py:8]: I think the import you do `sonrai.platform.aws.arn` have the utility function parse the id.  May to re-use if possible? 

https://github.com/sonraisecurity/sonrai-bots/pull/47/files

Comment (maxclaytoncary) [reply remediation/aws/enable_versioning_s3/bot.py:8]: When it parses the id, what is returned? I didn't make it very clear in the code but i'm getting the "name" of the resource out of the arn and passing that to the put_bucket_versioning as it didn't like the ARN.

Comment (wei-shang-sonrai) [reply remediation/aws/enable_versioning_s3/bot.py:8]: I think under the hook (https://github.com/sonraisecurity/remediation-bots/blob/master/sonrai/platform/aws/arn.py) it uses https://pypi.org/project/arnparse/

```
from arnparse import arnparse

arn = arnparse('arn:aws:s3:::my_corporate_bucket/exampleobject.png')

assert arn.partition == 'aws'
assert arn.service == 's3'
assert arn.region is None
assert arn.account_id is None
assert arn.resource_type is None
assert arn.resource == 'my_corporate_bucket/exampleobject.png'
```

so I am guessing `arn.resource` would be  what you you are looking for? 

In any case, if we have common parsing logic, we probably want to put them together somewhere. It doesn't have to be in this PR though.

Comment (maxclaytoncary) [reply remediation/aws/enable_versioning_s3/bot.py:8]: Ah okay gotchya - so did you want me to use arn.resource or is this okay for now? (I am completely new to python)

Comment (wei-shang-sonrai) [reply remediation/aws/enable_versioning_s3/bot.py:8]: I think gate keepers of this repo are going to review all the PRs from bot blitz, leave it as it is and see what they think.  Can't say I know a lot about python or bots.. :)

**Pull request merged**